syntax = "proto3";

package setter;
option java_package = "com.ahmad.grpc.demo.controller";

service Setter {

    rpc SetUser(UserRequest) returns(UserReply){}

}

message UserRequest{
    string firstName = 1;
    string lastName = 2;
    string dob = 3;
    string email = 4;
    string phone = 5;
    string updatedAt=6;
}

message UserReply{
   
}
